systemtests/tests/testfind/testrunner

```diff
@@ -20,23 +17,50 @@ zstat=0
 
 #shellcheck source=../scripts/functions
 . "${rscripts}"/functions
+"${rscripts}"/cleanup
+
 setup_data
 
-testfind_log=$tmp/testfind.out
+selftest_log=$tmp/selftest_log.out
+wildcard_log=$tmp/wildcard_log.out
+error_log=$logdir/bareos.log
+
+rm -f $selftest_log
+rm -f $wildcard_log
+rm -f $error_log
 
-rm -f $testfind_log
-./sbin/testfind-testfind -c etc/bareos | tee  "$testfind_log"
+./sbin/testfind-testfind -c etc/bareos -f SelfTest | tee  "$selftest_log"
```

Comment:
```suggestion
./sbin/testfind-testfind -c etc/bareos -f SelfTest > "$selftest_log"
```
Correct me if I am wrong here, but tests usually do not echo out what they are doing.